requirements.txt

```diff
@@ -0,0 +1,2 @@
+# generated from manifests external_dependencies
```

Comment:
@pedrobaeza Is this auto-generated? How?

Reply:
With pre-commit